deploy/kiali/kiali_cr.yaml

```diff
@@ -724,6 +724,10 @@ spec:
 #    ---
 #    istio_upgrade_action: false
 #
+# When there is no VirtualService referencing to a subset in a DestinationRule, skip that subset's validation.
+#    ---
+#    skip_subset_validation: false
+#
```

Comment:
We have a section here "validations" that is perfect for this kind of thing. Any settings that have to deal with validations (like this one) should go under there. So move this under `spec.kiali_feature_flags.validations`